skiko/src/commonMain/kotlin/org/jetbrains/skia/Expects.kt

```diff
@@ -14,7 +14,9 @@ expect class Pattern {
 }
 
 expect class Matcher {
+    @Deprecated("Not supported in k/n. Consider using group(ix: Int)")
```

Comment:
Let’s just remove it, but keep the comment.

Reply:
fixed